.github/workflows/main.yml

```diff
@@ -349,14 +299,17 @@ jobs:
     - run: |
         cargo build -p wasm-bindgen-cli
         ln -snf `pwd`/target/debug/wasm-bindgen $(dirname `which cargo`)/wasm-bindgen
-    - run: mv _package.json package.json && npm install && rm package.json
+    - run: echo 'packages:\n  - "examples/*"' > pnpm-workspace.yaml && sed -i -e 's/\\n/\n/g' pnpm-workspace.yaml
```

Comment:
As far as I understand it, what it is currently doing is just installing the top level `_package.json` dependencies, and when running the packages it just locates their dependencies one level up. Meaning before #4056 it was using `webpack-dev-server` 3 even if the packages specified 4. It works because all their dependencies are covered by the top package dependencies.
I added `-r` here to also recursively install dependencies of packages in subdirectories, and with the workspace we restrict it to the `examples` directory and allow them to share the same `node_modules`, basically intentionally creating the behavior of before, but actually resolving their listed dependencies. (You could say this is a "fix" and technically you don't need the top level `_package.json` anymore after this.)

Reply:
Unless you mean the use of `sed`, then sure. I didn't find a way to directly input newlines into the file - with `zsh` it just works to use `\n` but `bash` always escaped it. I'll add that to the comment too, unless you know a way to do this without `sed`.